mkdocs.yml

```diff
@@ -22,6 +22,9 @@ nav:
       - Blockchain:
           - Ethereum: topics/blockchain/ethereum.md
           - Features: topics/blockchain/features.md
+      - Sand:
+          - BundeSandSale: topics/sand/bundle-sand-sale.md
+  - Reference:
```

Comment:
`DeFi` is supposed to be within `topics`

Reply:
Fixed a merge issue, reference make no sense anymore, everything is in the topics folder